Zuntlan

supercow2 opened the pull request.

supercow2 committed:
Updating Zuntlan Development

Every zuntlan provence has been updated except for the one thixi provence that for some reason i can not find.

supercow2 committed:
Zuntlan

Everything in Zuntlan is finished except "capitals"

Lunu-Lunu committed:
fix some weird issues

Lunu-Lunu reviewed (changes requested):
dev trends a little too high, even for zuntlan's insane population- would recommend lowering the base production in most of the provinces to help model that zuntlan relies more on tax better. capital's dev is fine, though, keep that- just lower dev of the other provinces in zuntlan to make sure it isn't overly strong dev-wise

supercow2 committed:
Dev changes

removed around 15-20 dev and zuntlan is more tax based now

supercow2 committed:
Zuntlan

Finally, I have done development, please fucking accept this please my ego my ego MY EGO PLEASE
